### ISSUE TITLE
Can you check once if you are learning well?

### ISSUE THREAD

**Issue by TejNote** [NVIDIA/tacotron2#375]
Hi
I am currently learning Korean with Tacotron2.
The hparams are as follows.

epochs=500,
iters_per_checkpoint=1000,
seed=1234,
dynamic_loss_scaling=True,
fp16_run=False,
distributed_run=True,

dist_backend="nccl",
dist_url="tcp://localhost:54323",
cudnn_enabled=True,
cudnn_benchmark=False,
ignore_layers=['embedding.weight'],

################################
Data Parameters #
###############################
load_mel_from_disk=False,
training_files='filelists/kss_train_filelist_v3.txt',
validation_files='filelists/kss_val_filelist_v3.txt',
text_cleaners=['korean_cleaners'], # english_cleaners, korean_cleaners

################################
Audio Parameters #
################################
max_wav_value=32768.0,
sampling_rate=22050,
filter_length=1024,
hop_length=256,
win_length=1024,
n_mel_channels=80,
mel_fmin=0.0,
mel_fmax=8000.0,

################################
Model Parameters #
################################
n_symbols=len(kor_symbols),
symbols_embedding_dim=512,

encoder_kernel_size=5,
encoder_n_convolutions=3,
encoder_embedding_dim=512,

n_frames_per_step=1, # currently only 1 is supported
decoder_rnn_dim=1024,
prenet_dim=256,
max_decoder_steps=1000,
gate_threshold=0.5,
p_attention_dropout=0.1,
p_decoder_dropout=0.1,

attention_rnn_dim=1024,
attention_dim=128,

attention_location_n_filters=32,
attention_location_kernel_size=31,

postnet_embedding_dim=512,
postnet_kernel_size=5,
postnet_n_convolutions=5,

################################
Optimization Hyperparameters #
################################
use_saved_learning_rate=False,
learning_rate=1e-3,
weight_decay=1e-6,
grad_clip_thresh=1.0,
batch_size=32,
mask_padding=True # set model's padded outputs to padded values

And the alignment by learning is as follows.

![VIDEO_c35ce1a0-0369-438c-afc2-3efdc0e6162f](https://user-images.githubusercontent.com/16408262/84727156-65d58200-afc9-11ea-9686-a39c0a6b0d9b.gif)

But the very strange thing is that the waveform is strange between 0 and 50. In this case, did you fail to learn?

**Comment by ksaidin** [comment]
Try trimming your dataset by @Yeongtae 's [script](https://github.com/Yeongtae/tacotron2/blob/master/preprocess_audio.py). 

If you are training from scratch, I suggest to train with pretrained model first to see if your dataset is OK.

**Comment by TejNote** [comment]
@ksaidin 
Thank you!
There is one question I have told you.
Is it possible to learn to use the pre-trained model in other languages?

**Comment by ksaidin** [comment]
@TejNote yes. Make sure you follow [these steps](https://github.com/NVIDIA/tacotron2/issues/321#issuecomment-603894212).

**Comment by aishweta** [comment]
@TejNote yes you can train any language dataset using tacoton2 pre-tarined model. It learns faster way.

**Comment by TejNote** [comment]
![14000](https://user-images.githubusercontent.com/16408262/84984488-e507b900-b175-11ea-9b33-d8b9490753a1.png)
@ksaidin 
With your advice, it seems to be doing normal learning.
Thank you very much.
And as I told you, let's learn again with a pre-trained model.

@shwetagargade216 
Thank you for your advice.
Let's learn again with a pre-trained model.

**Comment by TejNote** [comment]
![individualImage](https://user-images.githubusercontent.com/16408262/85090728-d0342f80-b220-11ea-856f-dc0e799c607a.png)
@ksaidin @shwetagargade216 
I learned it the way you told me.
The alignment is very well formed.
I will share it once more when it becomes the 10k step.
